x/programs/rust/examples/token/src/lib.rs

```diff
@@ -1,22 +1,17 @@
-use wasmlanche_sdk::program::Program;
 use wasmlanche_sdk::store::Context;
-use wasmlanche_sdk::types::{Address, Bytes32};
+use wasmlanche_sdk::types::Address;
 
 use expose_macro::expose;
 
 /// Initializes the program with a name, symbol, and total supply.
 #[expose]
-pub fn init_program() -> i64 {
-    let mut token_program = Program::new();
-    token_program
-        .add_field(String::from("name"), Bytes32::from("WasmCoin".to_string()))
+pub fn init(ctx: Context) -> bool {
+    ctx.store_value("total_supply", &123456789_i64)
         .and_then(|_| {
-            token_program.add_field(String::from("symbol"), Bytes32::from("WACK".to_string()))
+            ctx.store_value("name", &String::from("WasmCoin"))
+                .and_then(|_| ctx.store_value("symbol", &String::from("WACK")))
```

Comment:
Change `store_value<T>` and `store_key_value<T>` to have `T: Serialize + ?Sized` and this will work.

```suggestion
            ctx.store_value("name", "WasmCoin")
                .and_then(|_| ctx.store_value("symbol", "WACK"))
```